apps/deploy-web/src/components/turnstile/Turnstile.tsx

```diff
@@ -40,7 +42,26 @@ export const Turnstile: FC = () => {
   const dismissedSubject = useRef(new Subject<void>());
 
   useEffect(() => {
-    return addResponseInterceptor(async error => {
+    if (originalFetch) {
+      window.fetch = async (...args) => {
```

Comment:
question(non-blocking): do we need a tech debt issue or this is kind of hack that must live with the code forever?